auv_sim/auv_sim_bridge/scripts/sim_bin_mock.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import rospy
+from std_srvs.srv import Trigger, TriggerResponse
+from gazebo_msgs.srv import SpawnModel, GetModelState
+from geometry_msgs.msg import Pose
+import rospkg
+import os
+
+
+def handle_drop_ball(req):
+    try:
+        rospy.wait_for_service("/gazebo/spawn_sdf_model")
+        rospy.wait_for_service("/gazebo/get_model_state")
+
+        spawn_model = rospy.ServiceProxy("/gazebo/spawn_sdf_model", SpawnModel)
+        get_model_state = rospy.ServiceProxy("/gazebo/get_model_state", GetModelState)
+
+        rospack = rospkg.RosPack()
+        pkg_path = rospack.get_path("auv_sim_description")
+
+        ball_models = [("ball_one.sdf", "sphere_one"), ("ball_two.sdf", "sphere_two")]
```

Comment:
see here, the names would be sphere_one and sphere_two, the sdf file can be a single one